apps/aehttp/Makefile

```diff
@@ -0,0 +1,14 @@
+SWAGGER_UI_GIT =https://github.com/swagger-api/swagger-ui.git
+SWAGGER_UI_GIT_DIR =./priv/swagger-ui
+SWAGGER_DOCS_DIR =./priv/swagger-docs
+
+.PHONY: swagger-docs
+swagger-docs: |$(SWAGGER_UI_GIT_DIR)/.git
```

Comment:
* Target `clean` nice to have.
* Targets `distclean` nice to have.
* Integration of target `distclean` in main Makefile nice to have.